trunner/config.py

```diff
@@ -39,7 +39,7 @@ def resolve_phrtos_dir() -> Path:
 DEVICE_TARGETS = ['armv7m7-imxrt106x']
 
 # Port to communicate with hardware board
-DEVICE_SERIAL = "/dev/ttyACM0"
+DEVICE_SERIAL = "/dev/mimxrt1064-debug-usb"
```

Comment:
I added "KERNEL=="ttyACM*", ATTRS{idVendor}=="0d28", ATTRS{idProduct}=="0204", SYMLINK+="mimxrt1064-debug-usb" to 10-local.rules file in /etc/udev/rules.d on both raspberry, because of differences in standard udev serial symlinks on both eval kits ("usb-ARM_DAPLink_CMSIS-DAP_0232000007b01b8800000000000000000000000097969905-if01" and "usb-ARM_DAPLink_CMSIS-DAP_023200001ce1efee00000000000000000000000097969905-if01"). 

Symlinks from by-path directory are different too:
platform-fd500000.pcie-pci-0000:01:00.0-usb-0:1.3:1.0-port0
platform-fd500000.pcie-pci-0000:01:00.0-usb-0:1.4:1.1

Reply:
- if we need to add specific UDEV rules for it to work, than it needs to be described on confluence
 - regarding the naming 
  - "by serial" being different is expected (they should have different serial)
  - "by path" should be the same if we use the same ports. Please investigate outputs of `lsusb -tv` indeed the same port is being used on every RPi